dm/templates/instance_template/instance_template.py.schema

```diff
@@ -20,7 +20,7 @@ info:
 
 required:
   - diskImage
-  - network
```

Comment:
Please update your implementation in a way that both 'network' (as the current implementation) as 'networks' (the new implementation) is supported to assure backwards compatibility.